german/Leichen/Leichen.md

```diff
@@ -5,4 +5,12 @@ werfen die beiden Leichen aus dem Boot.
 
 Sie schnaufen einmal kräftig durch während ihnen tausend Gedanken durch den Kopf gehen...
 
-Schnell, aber unbemerkt versuchen Sie an Land zu kommen, aber es scheint nicht näher zu kommen....
\ No newline at end of file
+Schnell, aber unbemerkt versuchen Sie an Land zu kommen, aber es scheint nicht näher zu kommen....
+
+Was tun Sie?
+
+Ich halte nach etwas (Ausschau)[Ausschau/Ausschau.md], an dass ich mich klammern könnte.
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.